python/lsst/cp/pipe/cpCombine.py

```diff
@@ -293,20 +289,20 @@ def run(self, inputExps, inputScales=None, inputDims=None):
         # Apply scaling:
         expScales = []
         if inputDims is None:
-            inputDims = [dict() for i in inputExps]
+            inputDims = [dict() for i in inputExpHandles]
 
-        for index, (exp, dims) in enumerate(zip(inputExps, inputDims)):
+        for index, (expHandle, dims) in enumerate(zip(inputExpHandles, inputDims)):
             scale = 1.0
-            if exp is None:
-                self.log.warning("Input %d is None (%s); unable to scale exp.", index, dims)
-                continue
-
+            visitInfo = expHandle.get(component='visitInfo')
```

Comment:
I won't mind if you choose to ignore this, as it seems that this file is already using a mixture of `'` and `"`, but it seems like `"` is preferred more often than not?

Reply:
Changed to more universally use `"`.  I will likely never be consistent, so I'm happy to fix these when they're pointed out.

Reply:
This is where black can really help: https://developer.lsst.io/python/formatting.html